compiler/grainformat/reformat.re

```diff
@@ -858,30 +963,45 @@ and print_pattern =
     | PPatOr(pattern1, pattern2) =>
       /* currently unsupported so just replace with the original source */
       let originalCode = get_original_code(pat.ppat_loc, original_source);
-      Walktree.remove_comments_in_ignore_block(pat.ppat_loc);
 
       (Doc.text(originalCode), false);
     | PPatAlias(pattern, loc) =>
       /* currently unsupported so just replace with the original source */
       let originalCode = get_original_code(pat.ppat_loc, original_source);
-      Walktree.remove_comments_in_ignore_block(pat.ppat_loc);
 
       (Doc.text(originalCode), false);
     };
 
   let (pattern, parens) = printed_pattern;
 
-  let with_leading =
-    if (leading_comment_docs == Doc.nil) {
-      [pattern];
-    } else {
-      [leading_comment_docs, Doc.space, pattern];
-    };
+  let with_leading = [pattern];
+
+  let after_parens_comments =
+    Comment_utils.get_comments_to_end_of_line(pat.ppat_loc, comments);
+  let after_parens_comments_docs =
+    Comment_utils.line_of_comments_to_doc_no_break(
+      ~offset=true,
+      after_parens_comments,
+    );
   let with_trailing =
-    if (trailing_comment_docs == Doc.nil) {
+    if (after_parens_comments_docs == Doc.nil) {
       with_leading;
     } else {
-      List.append(with_leading, [trailing_comment_docs]);
+      List.append(
+        with_leading,
+        [
+          after_parens_comments_docs,
+          // if (List.length(after_parens_comments) > 0) {
+          //   Doc.Text("bb");
+          // } else {
+          //   Doc.nil;
+          // },
```

Comment:
What's up with this?

Reply:
Well that's weird.   It was some layout debugging.